lib/model/QubitInformationObject.php

```diff
@@ -1582,6 +1582,37 @@ public function setActorByName($name, $options)
   }
 
   /**
+   * Import data from a <originalsloc> tag in EAD2002
+   *
```

Comment:
Can you explain your strategy in this function for fixing the bug in the comments? It seems like we're just removing the <p> tag and <addressline> tags and making them all just in a giant string with newlines? It wasn't immediately obvious to me.